<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSelector</class>
 <widget class="QDialog" name="frmSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 192, 0);</string>
       </property>
       <property name="text">
        <string>Investment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="tblSelected">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Selected elements</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="cmdUp">
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdRight">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdLeft">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdDown">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Elements</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Accept selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
